content/wasm-functions/http-trigger-reference.md

```diff
@@ -0,0 +1,37 @@
+title = "HTTP Trigger Reference"
+template = "functions_main"
+date = "2025-07-11T00:00:00Z"
+enable_shortcodes = true
+
+---
+- [HTTP Requests](#http-requests)
+  - [Request Headers](#request-headers)
+
+Fermyon Wasm Functions currently supports the [http trigger type](https://spinframework.dev/v3/http-trigger) for Spin applications.
```

Comment:
If "http" here is specifically referring to the trigger `type`, then monospace it (`http` trigger type); if it's referring to the protocol, capitalise it (HTTP trigger type).